docker/cassandra-env.sh

```diff
@@ -0,0 +1,311 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
what's the purpose of storing this file?
is it different from default?

Reply:
Yep. In case when we need to connect to C* using nodetool outside of C* ring - we need to disable local JMX. It can be done only with this file. If we don't need to connect to C* cluster - I'm going to delete env file.